source/HockeySDK.AndroidBindings/Additions/CrashManager.cs

```diff
@@ -41,6 +41,18 @@ public static void Register(global::Android.Content.Context context, string urlS
 			ConnectUnhandledExceptionEvents(listener);
 		}
 
+        public static void Initialize(global::Android.Content.Context context, string appIdentifier, global::HockeyApp.Android.CrashManagerListener listener)
```

Comment:
Your whitespace differs from used in this file. Tabs are used in near existing lines.

Reply:
Gosh, thx!!